sdks/python/apache_beam/examples/snippets/snippets.py

```diff
@@ -845,6 +845,21 @@ def filter_words(x):
   p.run().wait_until_finish()
 
 
+def model_textio_compressed(renames, expected):
+  """Using a Read Transform to read compressed text files."""
+  p = TestPipeline()
+
+  # [START model_textio_write_compressed]
+  lines = p | 'ReadFromText' >> beam.io.ReadFromText(
+      'gs://my_bucket/path/to/input-*.csv.gz',
```

Comment:
I think we should change that too, not necessarily in this PR. (I am assuming these are non-functional anyway. Because of the renaming to local files.)